src/test/java/de/zalando/aruha/nakadi/controller/CursorsControllerTest.java

```diff
@@ -110,6 +112,22 @@ public void whenBodyIsNotJsonThenBadRequest() throws Exception {
                 .andExpect(status().is(HttpStatus.BAD_REQUEST.value()));
     }
 
+    @Test
+    public void whenGetThenOK() throws Exception {
+        getCursors().andExpect(status().is(HttpStatus.OK.value()));
```

Comment:
You don't really check much with this test.
I think you can also mock cursorsCommitService. getSubscriptionCursors() and check that in request you get what you expected.